### ISSUE TITLE
Support Scala 2.12

### ISSUE THREAD

**Issue by Rydgel** [NET-A-PORTER/scala-uri#123]
Now that Scala 2.12 is officially out, it might be good to publish artefacts for that version too.

**Comment by theon** [comment]
So it is! I hadn't noticed it was out today. Thanks for raising this @Rydgel. I'll get a build out ASAP.

**Comment by theon** [comment]
Currently waiting on:

 - https://github.com/sirthias/parboiled2/issues/170
 - https://github.com/spray/spray-json/issues/204
 - https://github.com/scoverage/scalac-scoverage-plugin/issues/192 (looking like this will be pushed at the weekend, so I am happy to turn off test coverage for the meantime to get a release out and enable again after the weekend)

**Comment by Rydgel** [comment]
Nice, everyone is working on that, it shouldn't be too long :)

**Comment by theon** [comment]
@Rydgel  `0.4.16` of scala-uri is now pushed for 2.12.0: https://oss.sonatype.org/content/repositories/releases/com/netaporter/scala-uri_2.12/0.4.16/

It should make it to maven central shortly.

**Comment by Rydgel** [comment]
Thanks very much!

**Comment by druhinsagargoel** [comment]
I don't see spray-can, spray-caching, spray-client, spray-servlet support for Scala 2.12 on Maven. Is it possible to support these?

**Comment by theon** [comment]
@druhinsagargoel This is just the repo for scala-uri. You'll have to ask over at https://github.com/spray/spray about the spray repos, but I think it is unlikely spray will support 2.12 as it is no longer maintained and has been superseded by Akka HTTP. I'd recommend upgrading to akka-http, which does have 2.12 support.
